js/client-mainmenu.js

```diff
@@ -168,8 +168,13 @@
 
 		addPM: function (name, message, target) {
 			var userid = toUserid(name);
-			if (app.ignore[userid] && name.substr(0, 1) in {' ': 1, '+': 1, '!': 1, '✖': 1, '‽': 1}) return;
-
+			if (app.ignore[userid] && name.substr(0, 1) in {' ': 1, '+': 1, '!': 1, '✖': 1, '‽': 1}) {
+				if (!app.ignoreNotified) {
+					message = '/nonotify A message from ' + BattleLog.escapeHTML(name) + ' was ignored.';
+					name = '&';
```

Comment:
Should this just be from the user? Would make it easier to reply probably.

Although the app should also prevent you from replying to users you've ignored.

Reply:
It's mostly consistency. The blockpms notified thing on the server is from &, so I figured it should be the same here.

Reply:
Is it possible to PM users you've ignored? we should probably fix that

Reply:
The blockpms notification should probably be updated to also be from the user I think

Reply:
Fixed both.